src/keyboard-shortcuts.tsx

```diff
@@ -0,0 +1,21 @@
+import { t } from './services';
+
+
+
+const shortcutItem = (labelKey: string, shortcut: string) => {
+  return {
+    label: t(labelKey),
+    shortcut
+  }
+};
+
+export const commandsGeneral = [
+  shortcutItem('commands-label.create', 'Option + n'),
+  shortcutItem('commands-label.edit', 'Cmd + Option + e'),
+  shortcutItem('commands-label.save', 'Cmd + s'),
+  shortcutItem('commands-label.duplicate', 'Option + C'),
+  shortcutItem('commands-label.toggle-lists-detail-accordion', 'spacebar'),
+  shortcutItem('commands-label.expand-list-detail-accordions', 'Cmd  + Option + b'),
+  shortcutItem('commands-label.collapse-list-detail-accordions', 'Cmd  + Option + g'),
+  shortcutItem('commands-label.go-to-filter-pane', 'Cmd + s'),
+];
```

Comment:
Do we need to do anything special for non-Mac users?

Reply:
Inside the modal component, Mod and Cmd will be replaced with Alt and Option automatically
https://github.com/folio-org/stripes-components/blob/d3cf1280b26e2d313441d6069be2eec44215da9b/lib/KeyboardShortcutsModal/KeyboardShortcutsModal.js#L34